configure

```diff
@@ -732,15 +737,39 @@ else
 	fi;
 fi
 
-if [ -z "$systemdsystemunitdir" ]; then
-	if $PKG_CONFIG --exists systemd; then
-		systemdsystemunitdir=$($PKG_CONFIG --variable=systemdsystemunitdir systemd)
-	fi
+if [ ! "$systemd" = 0 ]; then
+    if ! $PKG_CONFIG --exists systemd ; then
+        if [ -n "$systemdsystemunitdir" ] || [ "$systemd" -eq 1 ]; then
```

Comment:
shouldn't this include sysusersdir as well then?

Reply:
I just kept it for legacy reasons since it was the only way to request systemd support but I can also add sysuserdir as check here.